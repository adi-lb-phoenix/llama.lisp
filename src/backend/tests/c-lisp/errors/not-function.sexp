(c-lisp
    (declare a int)
    (set a 5)
    (return a))
