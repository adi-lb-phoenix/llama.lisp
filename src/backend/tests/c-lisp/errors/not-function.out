CodegenError: Not a function: ['declare', 'a', 'int']
